browsers/duckduckgo.safariextension/js/tabManager.js

```diff
@@ -33,13 +33,13 @@ class TabManager {
      * 3. When we get a new main_frame request
      */
     create(tabData) {
-        console.log(`CREATE TAB: ${tabData.url}`)
         // when it's created from a message event, use tabData.message.currentURL, but
         // when it's created from a beforeNavigation event, message object won't exist, so use tabData.url:
         let url = tabData.message ? tabData.message.currentURL : tabData.url
         let createTabData = {url: url, id: tabManager.getTabId(tabData)}
         createTabData.target = tabData.target
         
+        console.log(`CREATE TAB: ${url}`)
```

Comment:
This was unrelated, but the log output was showing undefined since tabData.url isn't always defined

Reply:
Makes sense.